tla-io/src/test/scala/at/forsyte/apalache/io/quint/TestQuintEx.scala

```diff
@@ -172,4 +189,16 @@ class TestQuintEx extends AnyFunSuite {
   test("can convert builtin iuminus operator application") {
     assert(convert(Q.app("iuminus", Q._42)) == "-42")
   }
+
+  test("can convert builtin Set operator application") {
+    assert(convert(Q.app("Set", Q.s, Q.s, Q.s)) == """{"s", "s", "s"}""")
+  }
+
+  test("can convert builtin exists operator application") {
```

Comment:
Would it make sense to include (negative) tests for the 2 exceptional cases in `binaryBindingApp`?

Reply:
I couldn't decide on those. The logic gets a bit more complicated, but there is still only really one way that constructor can go right, and looking at the code it is pretty evident to me that every other way will go wrong and result in the error... 

But maybe it worth adding a test for that.